src/pages/auth/login.jsx

```diff
@@ -0,0 +1,113 @@
+import "./auth.css";
+import { Link, useNavigate } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+import { useAuth } from "../../contexts/index";
+import { AiOutlineEyeInvisible, AiFillEye } from "react-icons/ai";
+export const Login = () => {
+  const [showPassword, setShowPassword] = useState(false);
+  const { setIsLoggedIn, setUserDetails } = useAuth();
+  const navigate = useNavigate();
+  const [loginCredentials, setLoginCredentials] = useState({
+    email: "",
+    password: "",
+  });
+  const [errorMsg, setErrorMsg] = useState("");
+  const guestCredentials = {
+    email: "adarshbalika@gmail.com",
+    password: "adarshbalika",
+  };
+  const setDetails = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    setLoginCredentials({ ...loginCredentials, [name]: value });
+  };
+  const loginHandler = async (data) => {
+    try {
+      const response = await axios.post("/api/auth/login", data);
+      if (response.status === 200) {
+        setIsLoggedIn(true);
+        setUserDetails(response.data.foundUser);
+        localStorage.setItem("token", response.data.encodedToken);
+        navigate("/");
+      }
+    } catch (error) {
+      setErrorMsg("YOUR EMAIL IS NOT REGISTERED WITH US😔");
```

Comment:
you can give a alert box here, if the user wants to signup

Reply:
I'll keep this for now as I have the sign up link just below it, but definitely once done with everything I'll implement this